src/init.cpp

```diff
@@ -1434,6 +1430,11 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
     };
     ApplyArgsManOptions(args, mempool_opts);
     mempool_opts.check_ratio = std::clamp<int>(mempool_opts.check_ratio, 0, 1000000);
+
+    size_t descendant_limit_bytes = mempool_opts.limits.descendant_size_vbytes * 40;
+    if (mempool_opts.max_size_bytes < 0 || mempool_opts.max_size_bytes < descendant_limit_bytes) {
+        return InitError(strprintf(_("-maxmempool must be at least %d MB"), std::ceil(descendant_limit_bytes / 1000000.0)));
```

Comment:
7e90c69f1be6f8ba52e1f24b983379d00aad2f65: Maybe use `1'000'000.0` (not sure if that compiles?)

Reply:
It does! Fixed as of a956afedfe